samples/sensor/bme68x/sample.yaml

```diff
@@ -0,0 +1,32 @@
+sample:
+  name: BME68X Sensor Sample
+tests:
+  sample.sensor.bme68x:
+    depends_on: i2c
+    harness: sensor
+    integration_platforms:
+      - thingy91_nrf9160
+      - thingy91_nrf9160_ns
+      - thingy53_nrf5340_cpuapp
+      - thingy53_nrf5340_cpuapp_ns
+    platform_allow: >-
```

Comment:
I don't know the sample.yaml format very well, but I think it's possible to add simple tests that look for console output using only yaml.

Reply:
do you have an example for this?